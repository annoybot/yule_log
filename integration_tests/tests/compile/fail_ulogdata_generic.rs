use yule_log::ULogData;

#[derive(ULogData)]
pub struct GenericStruct<T> {
    value: T,
}

fn main() {}
