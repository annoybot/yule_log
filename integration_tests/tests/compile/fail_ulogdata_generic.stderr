error: ULogData derive cannot be used with generic structs or where clauses.
 --> tests/compile/fail_ulogdata_generic.rs:4:12
  |
4 | pub struct GenericStruct<T> {
  |            ^^^^^^^^^^^^^
